src/scripts/main.js

```diff
@@ -1,3 +1,440 @@
 'use strict';
 
-// write code here
+sortingTable();
+selectedRows();
+editingTable();
+createForm();
+fillForm();
+
+const messagesArray = [
+  {
+    title: 'ERROR',
+    description: 'Age must be from 18 to 90',
+    type: 'error',
+  },
+  {
+    title: 'WARNING',
+    description: 'Please input only whole numbers',
+    type: 'warning',
+  },
+  {
+    title: 'SUCCESS',
+    description: 'All changes have been made',
+    type: 'success',
+  },
+  {
+    title: 'WARNING',
+    description: 'Input value must not be empty',
+    type: 'warning',
+  },
+  {
+    title: 'ERROR',
+    description: 'The name must contain only letters and one space',
+    type: 'error',
+  },
+  {
+    title: 'WARNING',
+    description: 'Input value must be more then 4 symbols',
+    type: 'warning',
+  },
+  {
+    title: 'ERROR',
+    description: 'Please input only positive number',
+    type: 'error',
+  },
+  {
+    title: 'ERROR',
+    description: 'Input value must not contain letters or symbols',
+    type: 'error',
+  },
+  {
+    title: 'SUCCESS',
+    description: 'New employee added to table',
+    type: 'success',
+  },
+  {
+    title: 'ERROR',
+    description: 'Name length must be more than 3 characters',
+    type: 'error',
+  },
+  {
+    title: 'ERROR',
+    description: 'Office not defined',
+    type: 'error',
+  },
+  {
+    title: 'ERROR',
+    description: 'Each field is required',
+    type: 'error',
+  },
+];
+
+function sortingTable() {
+  const head = document.querySelector('thead');
+
+  head.querySelectorAll('th').forEach((item) => {
+    const spanElem = document.createElement('span');
+
+    spanElem.style.fontSize = '12px';
+    item.append(spanElem);
+  });
+
+  const arrCompare = [1, 1, 1, 1, 1];
+
+  head.addEventListener('click', (eventFunc) => {
+    const cellNumber = eventFunc.target.cellIndex;
+    const asc = ' \u25BC';
+    const desc = ' \u25B2';
+    const convert = (stringNumber) => {
+      return Number(stringNumber.toLocaleString().replace(/\D/g, ''));
+    };
+    const compareVariables = (aa, bb) => {
+      let a = aa.children[cellNumber].textContent.toUpperCase();
+      let b = bb.children[cellNumber].textContent.toUpperCase();
```

Comment:
Give your variables clearer names

```suggestion
      let firstNumber = aa.children[cellNumber].textContent.toUpperCase();
      let secondNumber = bb.children[cellNumber].textContent.toUpperCase();
```